.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
For this one, I think we can use the kubernetes bug report template directly for consistency - https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/bug-report.md

Reply:
Sounds good. I will swap it out shortly.